### ISSUE TITLE
Pascal VOC is very low in precision 

### ISSUE THREAD

**Issue by XuanheLiu** [qqwweee/keras-yolo3#168]
With darknet53_weights.h5 training, loss can't come down, about loss30, val_loss46 or so.

**Comment by XuanheLiu** [comment]
Nothing can be detected

**Comment by Hello526** [comment]
您好，请问您是否在PASCAL VOC2012上训练过YOLOV3，不知道它的mAP能达到多少？我查找了很多文献，都没有找到相关的内容。
